contrib/pg_stat_timing/pg_stat_timing.c

```diff
@@ -0,0 +1,140 @@
+#include "tenbase/pg.h"
+#include "tenbase/fmgr.h"
+#include "tenbase/executor.h"
+#include "tenbase/utils/timestamp.h"
+#include "tenbase/utils/guc.h"
+#include "tenbase/catalog/pg_type.h"
+#include "tenbase/pgstat.h"
+
+PG_MODULE_MAGIC;
+
+void _PG_init(void);
+void _PG_fini(void);
+static ExecutorStart_hook_type prev_ExecutorStart_hook = NULL;
+static ExecutorEnd_hook_type prev_ExecutorEnd_hook = NULL;
+
+typedef struct {
+    TimestampTz start_time;
+    TimestampTz end_time;
+    int expected_time; // 预期时间（微秒）
```

Comment:
It is good to see that your code comments are detailed and clear, except that it is in Chinese. We hope you could comment codes in English as possible.

The style of your codes and comments are not that consistent with our codes. Below are the styles we want to see.
```
/* Init the hash table */
static void
init_query_timings(void)
{
	HASHCTL ctl;
	MemSet(&ctl, 0, sizeof(ctl));
	ctl.keysize = sizeof(uint32);
	ctl.entrysize = sizeof(QueryTiming);
	query_timings = hash_create("QueryTimings", 256, &ctl, HASH_ELEM | HASH_BLOBS);
}
```